### ISSUE TITLE
model download url not accesible

### ISSUE THREAD

**Issue by lukhy** [PaddlePaddle/DeepSpeech#316]
```
Download language model ...
download_lm_ch.sh: 8: [: 29e02312deb2e59b3c8686c7966d4fe3: unexpected operator
--2019-03-31 05:24:42--  http://cloud.dlnel.org/filepub/?uuid=5cd1688e-78d9-4b9e-9c2f-6f104bd5b518
Resolving cloud.dlnel.org (cloud.dlnel.org)... 180.76.189.142
Connecting to cloud.dlnel.org (cloud.dlnel.org)|180.76.189.142|:80... connected.
HTTP request sent, awaiting response... 302 Moved Temporarily
Location: http://domainwall.cloud.baidu.com/block.html [following]
--2019-03-31 05:24:42--  http://domainwall.cloud.baidu.com/block.html
Resolving domainwall.cloud.baidu.com (domainwall.cloud.baidu.com)... 180.149.133.169
Connecting to domainwall.cloud.baidu.com (domainwall.cloud.baidu.com)|180.149.133.169|:80... connected.
HTTP request sent, awaiting response... 403 Forbidden
2019-03-31 05:24:42 ERROR 403: Forbidden.
```

**Comment by aaronlyt** [comment]
+1

**Comment by huntzhan** [comment]
@pkuyym 
Hi team, could you help recover the download links? Thanks!

**Comment by code-R** [comment]
@kuke @luotao1 any help is really appreciated here..  we will push the released models to s3..since all the links are broken.

**Comment by kuke** [comment]
https://deepspeech.bj.bcebos.com/zh_lm/zh_giga.no_cna_cmn.prune01244.klm  (small)
https://deepspeech.bj.bcebos.com/zh_lm/zhidao_giga.klm (large)

Can you try this two urls?

**Comment by huntzhan** [comment]
@kuke Thanks a lot! By the way, could you help push the speech models as well?

**Comment by rhamnett** [comment]
This just the chinese ones? What about en?

**Comment by rhamnett** [comment]
Also would it be at all possible to get the checkpoint download for baidu8ken this is super important for people looking to do telephone training with transfer learning? Many thanks

**Comment by code-R** [comment]
@kuke thank you so much for providing the language model for mandarin..Can you please provide the acoustic models as well?
@huntzhan the english model link works for me.. you can download from mozilla deepspeech repo as well.

**Comment by huntzhan** [comment]
@code-R but the link of the Chinese model (BaiduCN1.2k Model) is still failing.

**Comment by code-R** [comment]
@huntzhan yeah that is the speech model.. I would like to try the BaiduCN1.2k model myself as well, let's hope @kuke provides that :)

**Comment by huntzhan** [comment]
hi @kuke, could you help push the speech model when you have time?

**Comment by code-R** [comment]
everyone.. for anyone looking for baidu_8k english model.. please use this link for download

**Comment by wshhja** [comment]
Is there anybody train the speech model by self successfully?

**Comment by huntzhan** [comment]
@code-R Someone sent me the mandarin pertained model. But, unfortunately, It's not acting properly as mentioned in #94 .

**Comment by wshhja** [comment]
> @code-R Someone sent me the mandarin pertained model. But, unfortunately, It's not acting properly as mentioned in #94 .

Excuse me,couldn't you send the speech model to me?My email address is awm24@qq.com

**Comment by nijil** [comment]
@huntzhan Did you get hold of the mandarin acoustic model? Could you please share a link here?

**Comment by kuke** [comment]
@lukhy @aaronlyt @huntzhan @code-R @wshhja @nijil All urls have been updated. They should be able to work now.
